### ISSUE TITLE
An invite with a transfer-point tx that has failed shows as confirmed

### ISSUE THREAD

**Issue by yosoyubik** [urbit/bridge#705]
This is a UI issue, but also related to the invite creation, since only the transfer-point tx failed to verify:

`[%verify-sig-and-nonce %failed from=[ship=~falsec-topner proxy=%own] [%transfer-point address=0x2477.5d73.c264.250d.bc7b.68a7.ccb7.bb4c.51be.9b57 reset=%.n]]`

**Comment by tomholford** [comment]
Able to see the `failed` TX on my side; are you able to see the same when changing the filter dropdown in the top right?

![image](https://user-images.githubusercontent.com/16504501/140201234-e7207a1a-dca9-49d6-8122-f7a740893fca.png)

**Comment by yosoyubik** [comment]
Yes, that works, but I think that the "summary status" (Planet Spawn: confirmed) should say something else (warning?) when one of the inner txs fails, specially the transfer since that's the one that creates the invite's master ticket.

**Comment by yosoyubik** [comment]
Since the transfer failed, the point is still owned by the star, so we could add an option to "regenerate" the invite for this specific point
<img width="269" alt="Screen Shot 2021-11-04 at 10 52 54 AM" src="https://user-images.githubusercontent.com/1033194/140293491-4afbaf4f-339d-48b5-a400-1bf297cdc494.png">

**Comment by yosoyubik** [comment]
The failed transfer could be also related to this: https://github.com/urbit/bridge/issues/724

Fix for it here: https://github.com/urbit/urbit/commit/e54805893f0f48d8fc62642f4311fe6efc534913

**Comment by willbach** [comment]
Moving this to Backlog because this can be reproduced and there is a fix for what might have caused it.

**Comment by nerveharp** [comment]
> Yes, that works, but I think that the "summary status" (Planet Spawn: confirmed) should say something else (warning?) when one of the inner txs fails, specially the transfer since that's the one that creates the invite's master ticket.

Bumping this to @jyng to consider. Each planet spawn consists of 4 transactions; all 4 may or may not make it through. I think we want to explicitly surface the status of each transaction, i.e. 1/4, 2/4, 3/4, completed.

**Comment by willbach** [comment]
@nerveharp we do surface the status of each transaction in the transaction history. We may want to additionally give the option to resend a failed transaction and exclude any invite codes that have not had all 3 post-spawn transactions go through

**Comment by nerveharp** [comment]
Re-bumping to @jyng to consider resending partially-complete invite transactions.
